### PR TITLE
SBH-74 Simplify API

### PR DESCRIPTION
**Pull request by PapaBravo**

### CODE REVIEW

File: stores-api.yaml [hunk 1]
```diff
@@ -609,17 +621,11 @@ components:
         - Kreuzberg
```
Comment (cnasenberg) [inline stores-api.yaml]: Let's remove the neighborhoods? :)

File: stores-api.yaml [hunk 2]
```diff
@@ -560,8 +567,6 @@ components:
           type: string
         contactTwitter:
           type: string
-        profileImageURL:
```
Comment (haugthom) [inline stores-api.yaml:563]: What is the new field for uploading the image?

File: stores-api.yaml [hunk 3]
```diff
@@ -605,21 +610,11 @@ components:
...
-        - Neukölln
-        - Körtekiez
-        geoLocation:
-          type: Point
-          coordinates:
-          - 13.408134
-          - 52.489908
-        profileImageURL: https://7cbf5172f31e91cb7a5d-d8e96be60ea1fd7ad98fb07a482142ee.ssl.cf3.rackcdn.com/store-front-X8l4AvUY.jpg
         profileVideoURL: https://youtu.be/PBkb2mzbqKE
-        shopOwnerImage: https://drive.google.com/file/d/1Ls5Yjc760oEfnibC9Xv29OtALtmV_Ss4/view?usp=sharing
```
Comment (haugthom) [inline stores-api.yaml:619]: What is the new field for uploading the image?

File: stores-api.yaml [hunk 4]
```diff
@@ -605,21 +610,11 @@ components:
...
-        neighborhood:
-        - Kreuzberg
-        - Neukölln
-        - Körtekiez
-        geoLocation:
-          type: Point
-          coordinates:
-          - 13.408134
-          - 52.489908
-        profileImageURL: https://7cbf5172f31e91cb7a5d-d8e96be60ea1fd7ad98fb07a482142ee.ssl.cf3.rackcdn.com/store-front-X8l4AvUY.jpg
```
Comment (haugthom) [inline stores-api.yaml:617]: What is the new field for uploading the image?

File: stores-api.yaml [hunk 5]
```diff
@@ -570,9 +575,9 @@ components:
           type: boolean
         hasShipping:
           type: boolean
-        shopOwnerImage:
```
Comment (haugthom) [inline stores-api.yaml:573]: What is the new field for uploading the image?

**Review by haugthom — CHANGES_REQUESTED** [review]

File: stores-api.yaml (hunk 3 above)
Comment (PapaBravo) [reply stores-api.yaml:619]: You upload via POST stores/{id}/image

**Review by haugthom — APPROVED** [review]

**Review by cnasenberg — APPROVED** [review]

**Pull request merged**